messaging/src/main/java/org/axonframework/queryhandling/SimpleQueryUpdateEmitter.java

```diff
@@ -340,10 +343,24 @@ public Builder updateMessageMonitor(
          * Sets the {@link SpanFactory} implementation to use for providing tracing capabilities. Defaults to a
          * {@link NoOpSpanFactory} by default, which provides no tracing capabilities.
          *
-         * @param spanFactory The {@link SpanFactory} implementation
+         * @param spanFactory The {@link SpanFactory} implementation.
          * @return The current Builder instance, for fluent interfacing.
+         * @deprecated Use {@link #spanFactory(QueryUpdateEmitterSpanFactory)}  instead as it provides more configurability.
          */
+        @Deprecated
```

Comment:
You didn't change the JavaDoc of the `Builder` class and `builder()` method in accordance with this change.

Reply:
Adjusted